hedera-node/hedera-schedule-service-impl/src/test/java/com/hedera/node/app/service/schedule/impl/test/handlers/ScheduleDeleteHandlerTest.java

```diff
@@ -83,37 +79,39 @@ void scheduleDeleteHappyPath() throws DecoderException {
     }
 
     @Test
-    // when schedule id to delete is not found, fail with INVALID_SCHEDULE_ID
+        // when schedule id to delete is not found, fail with INVALID_SCHEDULE_ID
```

Comment:
Whitespace? A couple instances of this. Definitely not a big deal, just wondered if it's intentional

Reply:
Spotless likes to add extra indentation to line comments.